release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1280,10 +1280,25 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 [id="ocp-release-note-cloud-cred-operator-bug-fixes_{context}"]
 ==== Cloud Credential Operator
 
+* Previously, the `aws-sdk-go-v2` software development kit (SDK) failed to authenticate an `AssumeRoleWithWebIdentity` API operation on an {aws-first} {sts-first} cluster. With this release, `pod-identity-webhook` now includes a default region so that this issue no longer persists. (link:https://issues.redhat.com/browse/OCPBUGS-45937[*OCPBUGS-45937*])
+
+* Previously, secrets in the cluster were fetched in a single call. When there were a large number of secrets, this caused the API to time out. With this release, the Cloud Credential Operator fetches secrets in batches limited to 100 secrets. This change prevents timeouts when there are large number of secrets in the cluster. (link:https://issues.redhat.com/browse/OCPBUGS-39531[*OCPBUGS-39531*])
+
+[discrete]
+[id="ocp-release-note-cluster-override-admin-operator-bug-fixes_{context}"]
+==== Cluster Resource Override Admission Operator
+
+* Previously, if you specified the `forceSelinuxRelabel` field in a `ClusterResourceOverride` custom resource (CR), and then modified it afterwards, the change would not be reflected in the `clusterresourceoverride-configuration`config map, which is used to apply the SELinux re-labeling workaround feature. With this update, the Cluster Resource Override Operator can track the change to the `forceSelinuxRelabel` feature in order to reconcile the config map object. As a result, the config map object is correctly updated when you change the `ClusterResourceOverride` CR field. (link:https://issues.redhat.com/browse/OCPBUGS-48692[*OCPBUGS-48692*])
+
+
 [discrete]
 [id="ocp-release-note-cluster-version-operator-bug-fixes_{context}"]
 ==== Cluster Version Operator
 
+* Previously, a custom Security Context Constraint (SCC) impacted any pod that was generated by the Cluster Version Operator from receiving a cluster version upgrade. With this release, {product-title} now sets a default SCC to each pod, so that any custom SCC created does not impact a pod.  (link:https://issues.redhat.com/browse/OCPBUGS-46410[*OCPBUGS-46410*])
```

Comment:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#security-context-constraints-scc
```suggestion
* Previously, a custom security context constraint (SCC) impacted any pod that was generated by the Cluster Version Operator from receiving a cluster version upgrade. With this release, {product-title} now sets a default SCC to each pod, so that any custom SCC created does not impact a pod.  (link:https://issues.redhat.com/browse/OCPBUGS-46410[*OCPBUGS-46410*])
```